build/patches/Disable-conversion-measurement-api.patch

```diff
@@ -0,0 +1,333 @@
+@@ -22,8 +22,6 @@ public class AttributionReporterImpl extends AttributionReporter {
+     public void reportAttributionForCurrentNavigation(WebContents webContents,
+             String sourcePackageName, String sourceEventId, String destination, String reportTo,
+             long expiry) {
+-        AttributionReporterImplJni.get().reportAttributionForCurrentNavigation(
+-                webContents, sourcePackageName, sourceEventId, destination, reportTo, expiry);
+     }
+ 
+     /**
+@@ -32,8 +30,6 @@ public class AttributionReporterImpl extends AttributionReporter {
+     @Override
+     public void reportAppImpression(BrowserContextHandle browserContext, String sourcePackageName,
+             String sourceEventId, String destination, String reportTo, long expiry) {
+-        AttributionReporterImplJni.get().reportAppImpression(
+-                browserContext, sourcePackageName, sourceEventId, destination, reportTo, expiry);
+     }
+ 
+     @NativeMethods
+diff --git a/content/public/browser/navigation_controller.cc b/content/public/browser/navigation_controller.cc
+--- a/content/public/browser/navigation_controller.cc
++++ b/content/public/browser/navigation_controller.cc
+@@ -36,7 +36,6 @@ NavigationController::LoadURLParams::LoadURLParams(const OpenURLParams& input)
+       blob_url_loader_factory(input.blob_url_loader_factory),
+       href_translate(input.href_translate),
+       reload_type(input.reload_type),
+-      impression(input.impression),
+       is_pdf(input.is_pdf) {
+ #if DCHECK_IS_ON()
+   DCHECK(input.Valid());
+diff --git a/third_party/blink/common/features.cc b/third_party/blink/common/features.cc
+--- a/third_party/blink/common/features.cc
++++ b/third_party/blink/common/features.cc
+@@ -32,7 +32,7 @@ const base::Feature kCSSContainerQueries{"CSSContainerQueries",
+ 
+ // Controls whether the Conversion Measurement API infrastructure is enabled.
+ const base::Feature kConversionMeasurement{"ConversionMeasurement",
+-                                           base::FEATURE_ENABLED_BY_DEFAULT};
++                                           base::FEATURE_DISABLED_BY_DEFAULT};
+ 
+ const base::Feature kGMSCoreEmoji{"GMSCoreEmoji",
+                                   base::FEATURE_DISABLED_BY_DEFAULT};
+diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
+--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
++++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
+@@ -470,13 +470,11 @@
+     },
+     {
+       name: "ConversionMeasurement",
+-      origin_trial_feature_name: "ConversionMeasurement",
+-      origin_trial_allows_third_party: true,
+-      status: "experimental",
++      depends_on: ["DisabledForBromite"],
+     },
+     {
+       name: "ConversionMeasurementEventSources",
+-      status: "test",
++      depends_on: ["DisabledForBromite"],
```

Comment:
If you start using this, shouldn't the other patches also use this approach?

Reply:
in fact, I put the statement in the other patch (https://github.com/bromite/bromite/pull/1582)...
if you don't merge that it won't compile.
maybe it is the case to put it alone.

beware that once again I have not added it to the patch list